app/frontend/components/combobox/combobox.tsx

```diff
@@ -77,50 +77,52 @@ export const ComboBox = <TOption extends { id: string }>({
       disabled={isCreating || disabled}
       onChange={handleChange}
     >
-      <div className="relative mt-1">
-        <div className="relative w-full cursor-default overflow-hidden rounded bg-white text-left shadow-md focus:outline-none focus-visible:ring-2 focus-visible:ring-white focus-visible:ring-opacity-75 focus-visible:ring-offset-2 focus-visible:ring-offset-teal-300 sm:text-sm">
+      <div className="mt-1">
+        <div className="flex w-full cursor-default justify-between overflow-hidden rounded rounded border bg-transparent text-left shadow-md focus:outline-none focus-visible:ring-2 focus-visible:ring-white focus-visible:ring-opacity-75 focus-visible:ring-offset-2 focus-visible:ring-offset-teal-300 dark:border-white sm:text-sm">
```

Comment:
Looks good. But you have two `rounded` styles here. Maybe fix that before merging.